src/Resources/doc/index.md

```diff
@@ -163,6 +163,30 @@ class EmailerService
 }
 ```
 
+Using an external REST mjml compiler 
+------------------------------------
+If you don't want to install mjml locally, you can use a external rest mjml compiler:
+To do so, modify your configuration as foloww:
+```yaml
+....
+assoconnect_mjml:
```

Comment:
I think you have to set `assoconnect_mjml.mjml_compiler_service` to `assoconnect_mjml.rest_mjml_compiler_client` otherwise you'll still use the `Assoconnect\MJMLBundle\Compiler\MjmlCompiler`

Reply:
I’'ve rewritten that part of the doc, is it more understandable now ?